CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+
+### Request a preliminary review
+
+Create a pull request with the initial implementation, functionality, and styling, and mention the `@blackbaud/skyux-maintainers` to request a review. Make changes and request additional reviews as necessary. For [large changes that effect more than 50 files](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#pull-requests), submit changes in multiple small pull requests instead of one large pull request.
+
+Wait until after the initial review to add tests and documentation because implementation details may change.
+
+### Add tests
+
+Write tests for your feature. All code requires 100 percent [unit test](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#unit-tests) code coverage, and components also require [visual regression tests](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#visual-regression-tests) and [accessibility tests](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#accessibility-tests). In addition, create [test fixtures](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#test-fixtures) for any new components.
+
+### Request a final review
+
+1. Before requesting a final review, [squash all commits on your feature branch into one](#squashing-all-commits-into-one). This will consolidate all your work and include a well-formed commit message to be used in the changelog.
+1. After pushing the squashed commits, mention the `@blackbaud/skyux-maintainers` to review the complete contribution. We review the implementation, testing, functionality, styling, localization, and accessibility.
+
+#### Addressing review feedback
+
+1. Make required changes to the source code.
+1. Rerun all test suites.
+1. Create a "fixup" commit:
+
+```
+git commit --all --fixup HEAD
+git push
+```
+
+#### Updating the commit message
+
+Sometimes, the reviewer will ask that you reword your commit message. You can do so using the `--amend` flag:
+
+```
+git commit --amend
+git push --force-with-lease
+```
+
+_Note: If you need to ammend an earlier commit, you'll need to [squash all commits into one](#squashing-all-commits-into-one)._
+
+#### Squashing all commits into one
+
+If your pull request branch has several non-fixup commits, you'll need to squash all of the commits into one commit with a message that meets the [`conventional-changelog` specification](https://github.com/conventional-changelog/conventional-changelog).
+
+Run the following command to squash all commits in your feature branch with a new commit message that you provide. While local changes will remain intact, it does rewrite your feature branch's commit history.
+
+```
+npm run dev:pristine
+```
+
+## <a name="coding-rules"></a> Coding Rules
+
+1. All features and bug fixes should include at least one test spec and should not lower the overall code coverage percentage.
```

Comment:
```suggestion
1. All features and bug fixes must include at least one test spec and must not lower the overall code coverage percentage.
```
I assume this is a requirement rather than a suggestion, but if it's optional, I'd stick with `should`.